subprojects/packagefiles/gee/gee/meson.build

```diff
@@ -106,3 +106,4 @@ gee_dep = declare_dependency(
     dependencies: gee_deps,
     include_directories: include_directories('.')
 )
+meson.override_dependency('gee-0.8', gee_dep)
```

Comment:
This is not strictly speaking needed. The dependency lookup uses the value in the .wrap file. `override_dependency` is only used in special cases (e.g. when package `foo` provides an unrelated dependency `bar` and you want that to be used as opposed to the original library that provides `bar`).

Reply:
> Not sure how that passed the CI, I thought we were checking for that.

Ah! missing from releases.json, so it did not get tested at all! Which explains why locally this wrap fails immediately with `ERROR: Target 'gee-0.8' has 1 outputs: ['libgee-0.8.a'], but only 4 "install_dir"s were found.`.